src/k8s/subscriptions/DEV-CSTAR/terraform.tfvars

```diff
@@ -305,6 +305,11 @@ configmaps_fainvoicemanager = {
   POSTGRES_POOLSIZE                                      = "2"
   POSTGRES_SHOW_SQL                                      = "true"
   LOG_LEVEL_FA_INVOICE_MANAGER                           = "DEBUG"
+  MS_AGENZIA_ENTRATE_HOST                                = "cstariobackendtest"
```

Comment:
Could you please link the application property file where these variables are used? (A permanent link from github would be fine)

Reply:
@GiovanniMancini done, hope it's clear now, thanks for your review